package.json

```diff
@@ -13,7 +13,9 @@
         "start": "nuxi start",
         "test:pinia": "vitest",
         "test:pinia:coverage": "vitest run --coverage",
-        "test:e2e": "cypress open --e2e --browser chrome --port 9090"
+        "test:e2e": "cypress open --e2e --browser chrome --port 9090",
+        "test:e2e:run": "cypress run --e2e --browser chrome",
```

Comment:
Should we just replace the other command with run or do we need both of them separate?

Reply:
Replacing makes sense. In general I try not to replace scripts in case I break something, but it sounds like the only place to fix is the github action